### DISCUSSION TITLE
Linux ppc64le support back planning or to know the possibilities.

### DISCUSSION THREAD

**Discussion by Pratikt2312** [vercel/turborepo#9130] [Help] [answered]
### Summary

Hi Team,
We just want to know if there are any plans to add ppc64le support again and if yes then when can we expect it. I have checked some issues related to this #5562 , #5957. We know you're in the process of moving to just Rust where supporting additional platforms should be more straight forward and consider adding support for additional platforms again when that's done.

Thanks.

**Comment by anthonyshew** [comment, marked as answer]
Our platform support policy is [here](https://turbo.build/repo/docs/getting-started/support-policy#platforms) and we don't have any plans to expand this support at the moment.
